lib/sms/sms_submit.c

```diff
@@ -40,7 +45,7 @@ LOG_MODULE_DECLARE(sms, CONFIG_SMS_LOG_LEVEL);
  * @param[in] encoded_number Number encoded into 3GPP format.
  * @param[out] encoded_number_size_octets Number of octets/bytes in encoded_number.
  *
- * @retval -EINVAL Invalid parameter.
+ * @retval -EINVAL Invalid parameter, that is, number is NULL or too long (over 20 characters).
```

Comment:
Number can not be `NULL`, there's an assert for that. Should it say, that the number is empty?

Reply:
Number can be NULL. It's checked in a different place than the empty one. I'll mention empty.

Reply:
And yes, NULL shouldn't be mentioned here.